corebehrt/functional/utils.py

```diff
@@ -92,6 +81,8 @@ def get_abspos_from_origin_point(
     timestamps: Union[pd.Series, List[datetime]], origin_point: datetime
 ) -> Union[pd.Series, List[float]]:
     """Get the absolute position in hours from the origin point"""
+    if isinstance(timestamps, dd.Series):  # len can cause issues with dask
+        return (timestamps - origin_point).dt.total_seconds() / 60 / 60
```

Comment:
I thought we wanted something less fine-grained than hours?

Reply:
That will be adressed in a future PR:)